test/test_misc.py

```diff
@@ -78,3 +80,10 @@ def test_documentation():
             message = there_are(problems, kind="problem", tense='past', show=False,
                                 context=f"found in the readthedocs declaration file, {_DCICUTILS_DOC_FILE!r}")
             raise AssertionError(message)
+
+
+def test_for_debugging_print():
+    confirm_no_uses(where=os.path.join(LIBRARY_DIR, "*.py"), patterns={
+        "call to print": "^[^#]*print[(]",
+        "active use of pdb.set_trace": "^[^#]*pdb[.]set_trace[(][)]"
+    })
```

Comment:
Might want a test to make sure an exception is raise if a print actually exists, e.g. I tried this:
`    import pytest
    import tempfile
    with tempfile.NamedTemporaryFile(dir=LIBRARY_DIR, suffix=".py", mode="w") as f:
        f.write("print('helloworld')")
        f.flush()
        with pytest.raises(RuntimeError):
            confirm_no_uses(where=os.path.join(LIBRARY_DIR, "*.py"), patterns={
                "call to print": "^[^#]*print[(]",
                "active use of pdb.set_trace": "^[^#]*pdb[.]set_trace[(][)]"
            })
    with tempfile.NamedTemporaryFile(dir=LIBRARY_DIR, suffix=".py", mode="w") as f:
        f.write("import pdb ; pdb.set_trace() ")
        f.flush()
        with pytest.raises(RuntimeError):
            confirm_no_uses(where=os.path.join(LIBRARY_DIR, "*.py"), patterns={
                "call to print": "^[^#]*print[(]",
                "active use of pdb.set_trace": "^[^#]*pdb[.]set_trace[(][)]"
            })
`

Reply:
I actually used the tool to eliminate a bunch of cases of both, but a test is a good idea, of course. Thanks for prototyping things. I'll see about putting something like that in.

Reply:
I added some tests.